package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/autorest.typescript",
-  "version": "4.2.4",
+  "version": "4.4.0",
```

Comment:
Had to make it 4.3.1 since 4.3.0 failed with 
![image](https://user-images.githubusercontent.com/21109913/84087522-2b536000-a99f-11ea-9e9e-2c728926901f.png)